src/command_frame.cc

```diff
@@ -3,13 +3,26 @@
 
 #include <cstdlib>  // EXIT_SUCCESS, EXIT_FAILURE
 #include <iostream> // std::cout
+#include <sstream>  // std::ostringstream
 
 #include "command.h"
 #include "utils.h"
 
-auto intelligence::jsonschema::cli::frame(
-    const std::span<const std::string> &arguments) -> int {
-  const auto options{parse_options(arguments, {})};
+static auto enum_to_string(const sourcemeta::jsontoolkit::ReferenceEntryType type) -> std::string {
+  switch (type) {
+  case sourcemeta::jsontoolkit::ReferenceEntryType::Resource:
+    return "Resource";
```

Comment:
Keep it all lowercase, like `resource` instead of `Resource`

Reply:
ookay... I did it because the source code had it Capitalized

Reply:
Yeah, different conventions on different languages!